### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by aljones15**
```
    verifyCapabilityInvocation
    Ed25519
      ✓ should verify a valid request (46ms)
      ✓ should verify a valid request with multiple expectedHosts
      ✓ should THROW if verificationMethod has been revoked
      ✓ should THROW if verificationMethod can not be framed
      ✓ should THROW if no getInvokedCapability
      ✓ should THROW if no documentLoader
      ✓ should THROW if there are no headers
      ✓ should THROW if keyId can not be dereferenced by the documentLoader
      ✓ should THROW if verificationMethod type is not supported
      ✓ should NOT verify unless both content-type and digest are set
      ✓ should NOT verify if there is no url
      ✓ should NOT verify if host is not in expectedHost
      ✓ should NOT verify if Signature is missing keyId
      ✓ should NOT verify if Signature is missing created
      ✓ should NOT verify if Signature is missing expires
      ✓ should NOT verify if there is no method
      ✓ should NOT verify if headers is missing host
      ✓ should NOT verify with additionalHeaders not used in Signature
      ✓ should NOT verify if headers is missing capability-invocation
    RSA
      ✓ should verify a valid request
      ✓ should verify a valid request with multiple expectedHosts
      ✓ should THROW if verificationMethod has been revoked
      ✓ should THROW if verificationMethod can not be framed
      ✓ should THROW if no getInvokedCapability
      ✓ should THROW if no documentLoader
      ✓ should THROW if there are no headers
      ✓ should THROW if keyId can not be dereferenced by the documentLoader
      ✓ should THROW if verificationMethod type is not supported
      ✓ should NOT verify unless both content-type and digest are set
      ✓ should NOT verify if there is no url
      ✓ should NOT verify if host is not in expectedHost
      ✓ should NOT verify if Signature is missing keyId
      ✓ should NOT verify if Signature is missing created
      ✓ should NOT verify if Signature is missing expires
      ✓ should NOT verify if there is no method
      ✓ should NOT verify if headers is missing host
      ✓ should NOT verify with additionalHeaders not used in Signature
      ✓ should NOT verify if headers is missing capability-invocation


  38 passing (3s)


=============================== Coverage summary ===============================
Statements   : 81.33% ( 61/75 )
Branches     : 73.91% ( 17/23 )
Functions    : 83.33% ( 5/6 )
Lines        : 81.33% ( 61/75 )
================================================================================
```

```
----------|----------|----------|----------|----------|-------------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------|----------|----------|----------|----------|-------------------|
All files |    81.33 |    73.91 |    83.33 |    81.33 |                   |
 index.js |      100 |      100 |      100 |      100 |                   |
 main.js  |    81.69 |    73.91 |      100 |    81.69 |... 01,102,103,128 |
 util.js  |       50 |      100 |       50 |       50 |                 5 |
----------|----------|----------|----------|----------|-------------------|
```

Additionally:

1. all tests are passing in karma
2. a travis ci script was added to the project

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        ]
+      },
+      node: {
+        Buffer: false,
+        process: false,
+        crypto: false,
+        setImmediate: false
+      },
+      externals: {
+        'bitcore-message': '\'bitcore-message\''
```
Comment (mattcollier) [inline karma.conf.js:58]: I'm guessing this file was copied from elsewhere, is this bitcore-message thing needed here?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - "8"
```
Comment (mattcollier) [inline .travis.yml]: Node 8 was EOL on December 31, 2019, can we remove it?

**Review by mattcollier — APPROVED** [review]
Couple comments, otherwise fantastic.  More test coverage!

File: karma.conf.js (hunk 1 above)
Comment (aljones15) [reply karma.conf.js:58]: it was copied for sure and no bitcore is not used here directly, but:

```
ls node_modules/bitcore-message
bower.json       dist  gulpfile.js  lib           README.md
CONTRIBUTING.md  docs  index.js     package.json  test
// it is there
```

and a grep for it in `node_modules` does show tons of results so some of the signature libraries still require it.
Removing it results in non-fatal karma errors:

```
ERROR in ./node_modules/bitcore-lib/lib/crypto/random.js
Module not found: Error: Can't resolve 'crypto' in '/work/ajones/digital_bazaar/http-signature-zcap-verify/node_modules/bitcore-lib/lib/crypto'
 @ ./node_modules/bitcore-lib/lib/crypto/random.js 15:15-32
 @ ./node_modules/bitcore-lib/index.js
 @ ./node_modules/bitcore-message/index.js
 @ ./node_modules/jsonld-signatures/lib/suites/EcdsaKoblitzSignature2016.js
 @ ./node_modules/jsonld-signatures/lib/suites.js
 @ ./node_modules/jsonld-signatures/lib/jsonld-signatures.js
 @ ./tests/10-verify.spec.js
```

so thanks for the check as it does appear it is still required.

File: .travis.yml (hunk 2 above)
Comment (aljones15) [reply .travis.yml]: @dlongley @gannan08  @davidlehn  can we get rather quick conesnsus on this: should travis files be upgraded to test:

```
node_js:
  - "10"
  - "12"
  - "13"
  - "node"
```

so to ensure we are testing only in currently supported node versions?

I think @mattcollier has a good point here.

Comment (dlongley) [reply .travis.yml]: +1 agree

Comment (aljones15) [reply .travis.yml]: Addressed here: https://github.com/digitalbazaar/http-signature-zcap-verify/pull/9/commits/a0480496eb65006c6f7bda967660ee32cf540272

I also did a quick test using nvm and there are no breaking changes from 10-13 so we are good to go post travis check.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - "10"
+  - "12"
+  - "13"
```
Comment (davidlehn) [inline .travis.yml:5]: I think tests should just be even numbered releases and the latest via "node" (which is "13" right now).  "13" will become unsupported and useless before "10" so less to update later if we just leave it out now.

**Pull request merged**